.travis.yml

```diff
@@ -10,5 +10,5 @@ before_install:
 script:
   - go test -v ./...
 after_success:
-  - gox -output "dist/{{.OS}}_{{.Arch}}_{{.Dir}}"
-  - ghr --username iangrunert --token $GITHUB_TOKEN --replace --prerelease --debug pre-release dist/
\ No newline at end of file
+  - scripts/build.sh
```

Comment:
I think this isn't working currently - I think it needs to be ./scripts/build.sh? Not sure.

https://travis-ci.org/iangrunert/git-ratchet/builds/69851021

You can see there's no output from calling scripts/build.sh